app/controllers/application_controller.rb

```diff
@@ -6,7 +6,8 @@ class ApplicationController < ActionController::Base
 
```

Comment:
Could you provide some written clarification about the distinction we're making between "locked" and "disabled" user accounts, and how we're communicating that outwards.

Reply:
Did this in `app/models/user.rb` by the data field.